sdk/src/test/tutorial/java/parity/FeatureLookthrough.java

```diff
@@ -0,0 +1,99 @@
+package parity;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+public class FeatureLookthrough {
```

Comment:
Rename to `FeatureExtractor` or something similar just to make it clear what its used for